client/state/action-types.js

```diff
@@ -67,6 +67,8 @@ export const COMMENTS_CHANGE_STATUS_SUCESS = 'COMMENTS_CHANGE_STATUS_SUCESS';
 export const COMMENTS_COUNT_INCREMENT = 'COMMENTS_COUNT_INCREMENT';
 export const COMMENTS_COUNT_RECEIVE = 'COMMENTS_COUNT_RECEIVE';
 export const COMMENTS_DELETE = 'COMMENTS_DELETE';
+export const COMMENTS_DELETE_FAILURE = 'COMMENTS_DELETE_FAILURE';
+export const COMMENTS_DELETE_SUCCESS = 'COMMENTS_DELETE_SUCCESS';
```

Comment:
Also, another question: if I remove them, then should I dispatch a simple `COMMENTS_DELETE` in the `onSuccess` and `onFailure` of the `http` dispatch?